api/tests/patientCreate.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+curl localhost:3333
+echo
+source login.sh
+oldKey=$key
+echo -en '\nShould Succeed\n'
+loginWith "$(curlHeaders localhost:3333/patient -d "patientId=100000&deceased=false&patientNotes=Interted Notes")"
```

Comment:
Heh, I never explained the test functions. `loginWith` changes $key, your session key; useful only for apis that do that. Instead just use curlHeaders.